examples/shared_vpc/main.tf

```diff
@@ -39,33 +39,39 @@ provider "google-beta" {
 module "host-project" {
   source            = "../../"
   random_project_id = "true"
-  name              = "${var.host_project_name}"
-  org_id            = "${var.organization_id}"
-  billing_account   = "${var.billing_account}"
-  credentials_path  = "${local.credentials_file_path}"
+  name              = var.host_project_name
+  org_id            = var.organization_id
+  billing_account   = var.billing_account
+  credentials_path  = local.credentials_file_path
 }
 
 /******************************************
   Network Creation
  *****************************************/
 module "vpc" {
-  source  = "terraform-google-modules/network/google"
-  version = "0.6.0"
+  # TODO: Switch to released version once
+  # https://github.com/terraform-google-modules/terraform-google-network/pull/47
+  # is merged and released.  This is here to fix the `Error: Unsupported block
+  # type` on the `triggers` block in network's main.tf file.
+  #
+  # source  = "terraform-google-modules/network/google"
+  # version = "0.8.0"
+  source = "git::https://github.com/terraform-google-modules/terraform-google-network.git?ref=master"
```

Comment:
Can we pin this to a specific version? 0.12.x compatibility was introduced in https://github.com/terraform-google-modules/terraform-google-network/commit/7ba1b231e0662f4eaf264e5ad4a7e7ba977a40cd